packages/components/src/unit-control/utils.js

```diff
@@ -3,13 +3,21 @@
  */
 import { isEmpty } from 'lodash';
 
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Units of measurements. `a11yLabel` is used by screenreaders.
+ */
 export const CSS_UNITS = [
-	{ value: 'px', label: 'px', default: 0 },
-	{ value: '%', label: '%', default: 10 },
-	{ value: 'em', label: 'em', default: 0 },
-	{ value: 'rem', label: 'rem', default: 0 },
-	{ value: 'vw', label: 'vw', default: 10 },
-	{ value: 'vh', label: 'vh', default: 10 },
+	{ value: 'px', label: 'px', default: 0, a11yLabel: __( 'pixels' ) },
+	{ value: '%', label: '%', default: 10, a11yLabel: __( 'percent' ) },
+	{ value: 'em', label: 'em', default: 0, a11yLabel: 'ems' },
+	{ value: 'rem', label: 'rem', default: 0, a11yLabel: 'rems' },
+	{ value: 'vw', label: 'vw', default: 10, a11yLabel: 'viewport widths' },
+	{ value: 'vh', label: 'vh', default: 10, a11yLabel: 'viewport heights' },
```

Comment:
Yeah I struggled with the same thing. I just did a bunch of searches in the code base and the full text of these abbreviations are translated in other areas of the app so it's a good idea to translate. For the `ems` and `rems` I chose to use the `_x()` method to give the translators more context since those are not at all clear. Let me know what you think 👍  4727adda3726f53512a717daaa85e27f92e785dd

Reply:
I was unaware of `_x`. Very cool. I think matching the rest of the code base makes the most sense.